rust/worker/src/index/fulltext/types.rs

```diff
@@ -24,24 +26,27 @@ impl ChromaError for FullTextIndexError {
     }
 }
 
-pub(crate) struct FullTextIndexWriter {
+pub(crate) struct FullTextIndexWriter<'me> {
+    // We use this to implement updates which require read-then-write semantics.
```

Comment:
mega nit: also add this comment to MetadataIndexWriter (where we have the reader field)?